charts/airflow/values.yaml

```diff
@@ -857,6 +857,107 @@ workers:
   ##
   extraVolumes: []
 
+###################################
+## COMPONENT | Triggerer
+###################################
+triggerer:
+  ## if the airflow triggerer should be deployed
+  ##
+  enabled: false
```

Comment:
There is a question of if the triggerer should be deployed by default, and I am leaning towards yes, but with a warning in the CHANGELOG for users of `2.0.X` and `2.1.X`, and an update to the ["How to use a specific version of airflow?" docs](https://github.com/airflow-helm/charts/tree/main/charts/airflow#how-to-use-a-specific-version-of-airflow).

We don't deploy if `airflow.legacyCommands = true`, so we really only worry about users of `2.0.X` and `2.1.X`.

___NOTE:__ obviously if we add the triggerer, we will make the default version of airflow to be `2.2.X`_